src/main/device/service.ts

```diff
@@ -1,45 +1,63 @@
 import { aes } from '@internxt/lib';
 import { dialog } from 'electron';
 import fetch from 'electron-fetch';
+
 import os from 'os';
 import path from 'path';
-
+import { addProcessIssue } from '../background-processes/process-issues';
 import { getHeaders } from '../auth/service';
 import configStore from '../config';
 
 export type Device = { name: string; id: number; bucket: string };
 
-export async function getOrCreateDevice() {
-  async function createDevice(deviceName: string) {
-    return fetch(`${process.env.API_URL}/api/backup/deviceAsFolder`, {
-      method: 'POST',
-      headers: getHeaders(true),
-      body: JSON.stringify({ deviceName }),
-    });
-  }
-
-  async function tryToCreateDeviceWithDifferentNames(): Promise<Device> {
-    let res = await createDevice(os.hostname());
+const addUnknownDeviceIssue = (error: Error) => {
+  addProcessIssue({
+    errorName: 'UNKNOWN_DEVICE_NAME',
+    kind: 'LOCAL',
+    name: error.name,
+    action: 'GET_DEVICE_NAME_ERROR',
+    errorDetails: {
+      action: '',
+      message: error.message,
+      code: '',
+      stack: error.stack || '',
+    },
+    process: 'GENERAL',
+  });
+};
 
-    let i = 1;
+async function createDevice(deviceName: string) {
+  return fetch(`${process.env.API_URL}/api/backup/deviceAsFolder`, {
+    method: 'POST',
+    headers: getHeaders(true),
+    body: JSON.stringify({ deviceName }),
+  });
+}
 
-    while (res.status === 409 && i <= 10) {
-      res = await createDevice(`${os.hostname()} (${i})`);
-      i++;
-    }
+async function tryToCreateDeviceWithDifferentNames(): Promise<Device> {
+  let res = await createDevice(os.hostname());
 
-    if (!res.ok)
-      res = await createDevice(
-        `${os.hostname()} (${new Date().valueOf() % 1000})`
-      );
+  let i = 1;
 
-    if (res.ok) {
-      return res.json();
-    } else {
-      throw new Error('Could not create device trying different names');
-    }
+  while (res.status === 409 && i <= 10) {
+    res = await createDevice(`${os.hostname()} (${i})`);
+    i++;
   }
 
+  if (!res.ok)
+    res = await createDevice(
+      `${os.hostname()} (${new Date().valueOf() % 1000})`
+    );
+
+  if (res.ok) {
+    return res.json();
+  } else {
+    const error = new Error('Could not create device trying different names');
```

Comment:
Questions reading this error on the logs, that could arise:
- Which names have been tried? 
- How many?

Reply:
The operation is retried up to 10 times

Names that we try:
- os.hostname
- os.hostname + UNIX timestamp

Reply:
Okay, but maybe adding this feedback to the logs, even in a simple manner could help to understand which names were tried and why this failed. So is not my doubt but anyone's doubt when the time to provide support comes. 

For instance, if some user has a non-supported or not handled hostname by the app (strange characters), we should be able to have all the context on that and realize the issue in a simple manner. If that situation comes, without feedback on which names have been tried, we should need another release to see what is going on. So, in the case of the desktop platform, I suggest having more rich context even if it sounds obvious than having just a bit and not being able to determine what is happening, at least until we have a faster way of releasing and more flexibility to avoid some of these details